common/script_values/wc_men_at_arms_values.txt

```diff
@@ -26,6 +28,10 @@
 @massive_aerial_low_maint_cost = @[massive_aerial_recruitment_cost * recr_to_maint_factor]
 @massive_aerial_high_maint_cost = @[massive_aerial_low_maint_cost * high_maint_mult]
 
+@riflemen_recruitment_cost = 135
```

Comment:
I would...
```suggestion
@riflemen_recruitment_cost = 130
```
To make it more efficient so the AI will pick it.
Note: don't forget to change maint cost and `ai_quality`.